tensorflow_datasets/image/coco2017_panoptic.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
Would it be possible to add tests eventually ?

Reply:
I have to generate tests as per the documentation, right?

Reply:
Yes, please add tests - without them we'll not be able to detect issues with your code in the future.

Most of the work is actually in creating the fake data, the test file itself should be the easier part (please look at the documentation for details).

Reply:
Okay, I'll do that.

Reply:
@Conchylicultor Can you please help me ? I am getting this error when running test cases- 
```
.Downloading / extracting dataset coco2017 (?? GiB) to /var/folders/hr/dm5hrrkx1cq_h5kxx8h_k21c0000gn/T/coco2017_panoptic_testlW2lBH/tmpl1iUzS/coco2017/1.0.0...
E..s
======================================================================
ERROR: test_download_and_prepare_as_dataset (__main__.Coco2017_PANOPTIC_TEST)
test_download_and_prepare_as_dataset (__main__.Coco2017_PANOPTIC_TEST)
Run the decorated test method.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/absl/third_party/unittest3_backport/case.py", line 37, in testPartExecutor
    yield
  File "/Library/Python/2.7/site-packages/absl/third_party/unittest3_backport/case.py", line 162, in run
    testMethod()
  File "/Library/Python/2.7/site-packages/tensorflow_datasets/testing/test_utils.py", line 167, in decorated
    f(self, *args, **kwargs)
  File "/Library/Python/2.7/site-packages/tensorflow_datasets/testing/dataset_builder_testing.py", line 194, in test_download_and_prepare_as_dataset
    self._download_and_prepare_as_dataset(self.builder)
  File "/Library/Python/2.7/site-packages/tensorflow_datasets/testing/dataset_builder_testing.py", line 209, in _download_and_prepare_as_dataset
    builder.download_and_prepare(download_config=download_config)
  File "/Library/Python/2.7/site-packages/tensorflow_datasets/core/api_utils.py", line 52, in disallow_positional_args_dec
    return fn(*args, **kwargs)
  File "/Library/Python/2.7/site-packages/tensorflow_datasets/core/dataset_builder.py", line 219, in download_and_prepare
    max_examples_per_split=download_config.max_examples_per_split)
  File "/Library/Python/2.7/site-packages/tensorflow_datasets/core/dataset_builder.py", line 650, in _download_and_prepare
    for split_generator in self._split_generators(dl_manager):
  File "/Users/KJ/Desktop/GitHub/datasets_tf/tensorflow_datasets/image/coco2017_panoptic.py", line 97, in _split_generators
    image_dir=extracted_paths["train_images"],
KeyError: 'train_images'

----------------------------------------------------------------------
Ran 5 tests in 0.566s

FAILED (errors=1, skipped=1)
```

Reply:
You probably need to add `DL_EXTRACT_RESULT` in the test, like in https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/image/mnist_test.py

Reply:
@Conchylicultor Okay. I''l try it out.